### ISSUE TITLE
GH Actions automatic MSI builds are failing

### ISSUE THREAD

**Issue by ischoegl** [Cantera/cantera-windows-binaries#1]
MSI builder for the Matlab toolbox are currently failing due to 
```
cantera_shared.def : error LNK2001: unresolved external symbol <…>
```
Failures started in Fall 2022.

**Comment by speth** [comment]
After the fix introduced in Cantera/cantera#1505 for a different MSI build error, we're back to this one. For the sake of posterity, a complete build log is saved here: [x86-msi.log](https://github.com/Cantera/cantera-windows-binaries/files/11757965/x86-msi.log).

At present, only the 32-bit build is failing, and I think the relevant aspect is what was elided in the report above. All of the "unresolved external symbol" names are for decorated "C" methods, which includes the Sundials methods and the Cantera `clib` methods, e.g.:

```
cantera_shared.def : error LNK2001: unresolved external symbol _CVDlsSetLinearSolver
...
cantera_shared.def : error LNK2001: unresolved external symbol _thermo_meanMolecularWeight
...
```

[Documentation from Microsoft](https://learn.microsoft.com/en-us/cpp/build/reference/decorated-names?view=msvc-170#FormatC) indicates that there is a difference in how (or whether) "C" methods are decorated between 32 and 64 bit platforms.

**Suggested resolution**: Given that the main purpose of this package is to provide the Matlab toolbox, and that Mathworks stopped releasing 32-bit versions of Matlab for Windows in 2015, I think we can safely drop the 32-bit builds.

**Comment by ischoegl** [comment]
I agree that dropping support for 32-bit versions is an adequate resolution.
